database/config.go

```diff
@@ -7,14 +7,14 @@ import (
 
 // Config defines database client configuration.
 type Config struct {
-	Type       string     `yaml:"type" default:"mysql"`
-	Host       string     `yaml:"host"`
-	Port       int        `yaml:"port"`
-	Database   string     `yaml:"database"`
-	User       string     `yaml:"user"`
-	Password   string     `yaml:"password"`
+	Type       string     `yaml:"type" env:"TYPE" default:"mysql"`
+	Host       string     `yaml:"host" env:"HOST"`
+	Port       int        `yaml:"port" env:"PORT"`
+	Database   string     `yaml:"database" env:"DATABASE"`
+	User       string     `yaml:"user" env:"USER"`
+	Password   string     `yaml:"password" env:"PASSWORD,unset"`
```

Comment:
These environment names should have unique names indicating which config type they're referring to, e.g. this way you want to be able to distinguish `Host`, `PORT` etc. from their Redis counterparts.

Reply:
Nope, the application does it in its umbrella struct via envPrefix:"REDIS_"